src/Nethermind/Nethermind.Trie/MemoryAllowance.cs

```diff
@@ -22,6 +22,6 @@ public static class MemoryAllowance
     {
         public static long TrieNodeCacheMemory { get; set; } = 128.MB();
 
-        public static int TrieNodeCacheCount => (int)TrieNodeCacheMemory / PatriciaTree.OneNodeAvgMemoryEstimate;
+        public static long TrieNodeCacheCount => TrieNodeCacheMemory / PatriciaTree.OneNodeAvgMemoryEstimate;
```

Comment:
should not the fix be: (int)(TrieNodeCacheMemory / PatriciaTree.OneNodeAvgMemoryEstimate)
I would like to see the test showing how it was broken before and why it is fixed now

anyway, love the fact that a simple thing was fixed fast

Reply:
I see what you mean, but I thought that changing the type to long will allow bigger values, if (TrieNodeCacheMemory/PatriciaTree.OneNodeAvgMemoryEstimate) overflows int we would still have the error. What do you think would be better?